core/src/components/button/test/theme-ionic/button.e2e.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from '@playwright/test';
```

Comment:
I would recommend taking a look at: https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/button/test/size/button.e2e.ts to model the tests around. 

The tests will run significantly faster with a much smaller file size as a result.

Reply:
Done!